tests/format/typescript/literal/test.ts

```diff
@@ -0,0 +1,7 @@
+const fooBazBar1:string = 'Multiline string\
```

Comment:
This is not a literal type. See https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types